client/domain/FeedbackParagraphObjectFactory.js

```diff
@@ -121,6 +121,23 @@ tie.factory('FeedbackParagraphObjectFactory', [
       return new FeedbackParagraph(PARAGRAPH_TYPE_SYNTAX_ERROR, error);
     };
 
+    /**
+     * Returns a FeedbackParagraph created from an object.
+     *
+     * @param {Object} dict Object parsed from json
```

Comment:
The structure of this dict should be explained.

It seems a bit odd that we're reading what looks like private vars of the dict. One use of toDict() would be to change '_type' to 'type' and '_content' to 'content'.